app/Http/Controllers/User/VoteController.php

```diff
@@ -64,19 +64,46 @@ public function store(Request $request, $id)
             if ($vote = $this->vote->voteUp($review)) {
                 if ($vote->status == '0') {
                     $data = ['noup'];
+                    $this->notifiaction->destroy([
+                        'send_id' => Auth::id(),
+                        'receive_id' => $review->user_id,
+                        'target_type' => config('model.target_type.vote'),
+                        'target_id' => $vote->id,
+                    ]);
                 } else {
                     $data = ['up'];
+                    $this->notifiaction->store([
+                        'send_id' => Auth::id(),
+                        'receive_id' => $review->user_id,
+                        'target_type' => config('model.target_type.vote'),
+                        'target_id' => $vote->id,
+                        'viewed' => config('model.viewed.false'),
+                    ]);
                 }
                 $this->review->upVote($id, $data);
+                
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/User/VoteController.php</br>**Line**: 84</br>Blank line found at end of control structure